besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2764,7 +2765,8 @@ private String generateConfigurationOverview() {
     }
 
     if (profile != null) {
-      builder.setProfile(profile.toString());
+      // TODO: Do we need to convert _ to spaces?
```

Comment:
Is that for the profile name? I would say no, because that only make it harder. The name is the name!